RxCocoa/iOS/UITableView+Rx.swift

```diff
@@ -168,15 +168,6 @@ extension Reactive where Base: UITableView {
 extension UITableView {
  
     /**
-    Factory method that enables subclasses to implement their own `delegate`.
-    
-    - returns: Instance of delegate proxy that wraps `delegate`.
-    */
-    public override func createRxDelegateProxy() -> RxScrollViewDelegateProxy {
```

Comment:
Missed it, already fixed.
"override" now forbidden inside of extension, so I moved this logic into UIScrollView